### PR TITLE
Python script for mapping sample IDs:variants

### PR DESCRIPTION
**Pull request by Saakshi1997**
Do have a look at the code for mapping sample IDs in the variant call files to their labels ( probands and siblings )

### CODE REVIEW

File: mapping.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Script to map Sample IDs and obtain family ids + relationship
```
Comment (Saakshi1997) [inline mapping.py:1]: I have no clue why the script seems so shabby. Too many variables are being assigned and too many subsets of dataframes are made. Any Ideas?

File: mapping.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```
Comment (Saakshi1997) [inline mapping.sh:1]: Do have a look at the Slurm commands. I have a doubt about the ntasks commands. number of tasks ( What should it be in my case?
Am i taking up too many cpus or memory ? Need your help :(

File: mapping.py (hunk 1 above)
Comment (deeprob) [reply mapping.py:1]: 1. Use the names in pd.read_csv to correctly name "SFARI ID" and "Repository Id". When creating "Family ID" and "Relationship", use proper names without spaces. Then there is no need to rename columns later on. That eliminates lines 10 and 11.
2. Are you using df_grouped somewhere? I don't see it.

File: mapping.sh (hunk 2 above)
Comment (deeprob) [reply mapping.sh:1]: 1. ntasks should be one in you case. It's used for parallelizing within a single script.
2. Yes, your code is not parallelized at this level, so there is no point using 64 cpus, just use two. Memory is fine, I would suggest increasing it to 40G depending on the size of your individual chromosome files.

**Review by deeprob — APPROVED** [review]

**Pull request merged**